src/app/services/greenwave.service.spec.ts

```diff
@@ -43,7 +43,7 @@ describe('GreenwaveService testing', () => {
       'waivers': []
     };
 
-    greenwaveService.getDecision('some-continaer', 'cvp_default', 'cvp', 'koji_build').subscribe(data => {
+    greenwaveService.getDecision('some-continaer', 'cvp_default', 'cvp', 'redhat-container-image').subscribe(data => {
```

Comment:
i know this was here prior to the MR, but wondering if it's worth changing `some-continaer` to `some-container`